plugins/janus_streaming.c

```diff
@@ -2155,6 +2160,7 @@ int janus_streaming_init(janus_callbacks *callback, const char *config_path) {
 					continue;
 				}
 				gboolean is_private = priv && priv->value && janus_is_true(priv->value);
+				gboolean rtsp_quirk = quirk && quirk->value && janus_is_true(priv->value);
```

Comment:
This should be `janus_is_true(quirk->value)` (my typo in the original message, sorry)

Reply:
This feels less like black magic, with the correct value in there. =)